src/ansys/fluent/core/__init__.py

```diff
@@ -69,7 +69,9 @@ def version_info() -> str:
 USER_DATA_PATH = platformdirs.user_data_dir(
     appname="ansys_fluent_core", appauthor="Ansys"
 )
-EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")
+EXAMPLES_PATH = os.getcwd()
+
+PYFLUENT_CONTAINER_MOUNT_PATH = os.path.join(USER_DATA_PATH, "examples")
```

Comment:
```suggestion
EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")

CONTAINER_MOUNT_PATH = os.getcwd()
```

Reply:
And in `fluent_container.py`, instead of 
```
host_mount_path = pyfluent.EXAMPLES_PATH
``` 
it should be 
```
host_mount_path = pyfluent.CONTAINER_MOUNT_PATH
```

Reply:
Done. Thank you.